presentation/mypage/src/main/java/com/hilingual/presentation/mypage/component/SettingItem.kt

```diff
@@ -0,0 +1,126 @@
+package com.hilingual.presentation.mypage.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.core.designsystem.R as DesignSystemR
+
+@Immutable
+internal data class SettingItemData(
+    val iconRes: Int,
+    val title: String,
+    val trailing: SettingTrailing,
+    val onClick: (() -> Unit)?
+)
+
+internal sealed class SettingTrailing {
+    data object None : SettingTrailing()
+    data object Chevron : SettingTrailing()
+    data class Text(val value: String) : SettingTrailing()
+}
+
+@Composable
+internal fun SettingItem(
+    data: SettingItemData,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier
```

Comment:
```suggestion
        modifier = modifier
```
P2: 사소하지만 호옥시나 Row의 파라미터 순서가 변경되었을 때도 문제가 안생기려면 이렇게 해주는게 좋아요! 그리고 가독성도 좋아짐!!

Reply:
헉 그러게요 여기 modifier 써주는 걸 왜 빼먹었지... 추가했습니다! 👏🏻